src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -563,7 +563,8 @@ extension RiskProvider {
 				mostRecentDateWithHighRisk: risk.details.mostRecentDateWithRiskLevel,
 				numberOfDaysWithLowRisk: risk.details.numberOfDaysWithRiskLevel,
 				numberOfDaysWithHighRisk: 0,
-				calculationDate: Date(),
+				// setting 09:06 hours of same day
+				calculationDate: Calendar.utcCalendar.date(bySettingHour: 7, minute: 6, second: 0, of: Date()) ?? Date(),
```

Comment:
I think it makes sense to use the `current` calendar if you want to have "09:06" local time. This would not break when we switch back to winter time.

Reply:
Would it help to even use `autoupdatingCurrent` to prevent really weird (testing) issues?